src/fleet/azext_fleet/custom.py

```diff
@@ -189,25 +204,67 @@ def delete_fleet(cmd,  # pylint: disable=unused-argument
     return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, name, polling_interval=5)
 
 
-def get_credentials(cmd,  # pylint: disable=unused-argument
+def get_credentials(cmd,
                     client,
                     resource_group_name,
                     name,
-                    path=os.path.join(os.path.expanduser(
-                        '~'), '.kube', 'config'),
+                    path=os.path.join(os.path.expanduser('~'), '.kube', 'config'),
                     overwrite_existing=False,
-                    context_name=None):
-    credential_results = client.list_credentials(resource_group_name, name)
-    if not credential_results:
-        raise CLIError("No Kubernetes credentials found.")
+                    context_name=None,
+                    member_name=None):
+
+    # If a member name is given, we use the cluster resource ID from the fleet member
+    # to get that member cluster's credentials
+    # Otherwise, we get the credentials for the fleet hub
+    if member_name:
+        fleet_members_client = cf_fleet_members(cmd.cli_ctx)
+
+        try:
+            fleet_member = fleet_members_client.get(resource_group_name, name, member_name)
+            if not fleet_member:
+                raise CLIError(f"Fleet member '{member_name}' not found in fleet '{name}'.")
```

Comment:
this doesn't do anything - in the 404 case, you get an exception, and this doesn't catch it.  Can remove these lines.